deploy.env

```diff
@@ -144,5 +144,9 @@ APPFLOWY_INDEXER_REDIS_URL=redis://redis:6379
 APPFLOWY_COLLABORATE_MULTI_THREAD=false
 APPFLOWY_COLLABORATE_REMOVE_BATCH_SIZE=100
 
+# AppFlowy Worker
+APPFLOWY_WORKER_REDIS_URL=redis://redis:6379
+APPFLOWY_WORKER_DATABASE_URL=postgres://postgres:password@localhost:5432/postgres
```

Comment:
I would assume that it should have been `postgres` as opposed to localhost.

Reply:
I'm thinking we should just reuse the env var from appflowy-cloud. I cant think of any situation where it should be different